packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -10,7 +10,7 @@ meta:
 
 The Okta Administrator Roles API provides operations to manage administrative Role assignments for a User.
 
-Role listing APIs provide a union of both standard and Custom Roles assigned to a User or Group. We are, therefore, providing both groups of APIs and their intersections in this single doc.
+Role listing APIs provide a union of both standard and Custom Roles assigned to a User or Group. Okta provides both groups of APIs and their intersections in this doc.
```

Comment:
```suggestion
This document includes the operations of both APIs and where they intersect:
Roles: `api/v1/users|groups/{id}/roles*`
Custom Roles: `/api/v1/iam/*`
```

Reply:
This document includes the operations of _standard Roles and Custom Roles_ APIs and where they intersect: ... ?